tests/test_integration.py

```diff
@@ -1249,6 +1256,83 @@ async def test_info_unsuccessful_start(succeed_immediately):
     assert "failed to start" in stderr.lower(), stderr
 
 
+@pytest.mark.asyncio
+async def test_filter_given_implementations_lang(
+    envsonschema,
+    lintsonschema,
+    fakejsimpl,
+):
+    stdout, stderr = await bowtie(
+        "filter-implementations",
+        "-i",
+        envsonschema,
+        "-i",
+        lintsonschema,
+        "-i",
+        fakejsimpl,
+        "--supports-language",
+        "python",
+    )
+    output = {
+        "bowtie-integration-tests/envsonschema",
+        "bowtie-integration-tests/lintsonschema",
+    }
+    assert set(stdout.splitlines()) == output
```

Comment:
Don't use `set`, that could hide some strange bug where we print things out twice. Use `sorted` if you're just trying to avoid ordering problems.